docs/docs/deploy/heroku.md

```diff
@@ -0,0 +1,170 @@
+
+Our review app is live! This has it's own Heroku Postgres instance spun up so we can play around with it without impacting a shared database.
+
+### Step 5: Merging to master and deploying to production
+
+Now that we've done some QA on our review app, we'll merge the pull request and deploy to production. Normally, this is 
+where a teammate would review our change, but for now we can safely click the `Merge` button and carry on solo.
+
+Upon merging the pull request, the review app instance will be spun down - saving us some money/free dyno hours. At this point you can head back to the heroku pipeline to view the deploy logs.
+
+[![heroku deployment in progress](/assets/images/heroku-deploy/heroku-deploy-logs.png)](/assets/images/heroku-deploy/heroku-deploy-logs.png)
+
+Give it a couple of minutes to run and your app will be deployed! You can also check in on the deployment in the GitHub environment screen, by clicking the `environments` button on your repo's home page.
+
+[![github env is pending](/assets/images/heroku-deploy/github-env-pending.png)](/assets/images/heroku-deploy/github-env-pending.png)
+
+Once the app has been deployed this will change to read `Deployed`. 
+
+Click the `View deployment` button... and it's broken! This is because Heroku doesn't respect the `app.json` file's addons in production. I don't fully understand why they made this choice, but let's fix it.
+
+### Step 6: Spinning up the production database
+
+Navigate over to the application in Heroku and go to the resources tab:
+
+[![heroku app resources](/assets/images/heroku-deploy/heroku-resources.png)](/assets/images/heroku-deploy/heroku-resources.png)
+
+And provision a free database:
+
+[![Adding the heroku postgres addon](/assets/images/heroku-deploy/heroku-postgres-free.png)](/assets/images/heroku-deploy/heroku-postgres-free.png)
+
+Now click `More` at the top right and `Restart all dynos`. Now - after the restart completes - we can try clicking `Open App` again...
+
+[![Deployment was successful](/assets/images/heroku-deploy/deploy-success.png)](/assets/images/heroku-deploy/deploy-success.png)
+
+Success! Our app is now live in production!
+
+### Step 7: reflect on the result
+
+To conclude, we now have:
+
+* an automated process that runs all of our tests for a pull request;
+* on test success deploys the pull request code to a unique, isolated environment;
+* on merge deletes the review application automatically; and
+* deploys the change to production.
+
+This is incredibly powerful. Whether you're working solo or in a team, I think having a way to easily review and trial changes before 
+letting a customer get access to them is the way forward. There's more yet to do though! Continue on below to allow your application to send email,
+report errors to sentry and to have a long-lived staging environment for QA.
+
+## Adding mailgun
+
+TODO
+
+## Adding Staging
```

Comment:
inconsistent caps- "Staging" is capitalized but you've done lower case in the rest of the project